intellij/src/main/java/services/ProcessTransport.java

```diff
@@ -0,0 +1,89 @@
+package services;
+import com.github.arteam.simplejsonrpc.client.Transport;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+public class ProcessTransport implements Transport {
+    private final String[] processCommandsAndArgs;
+    public ProcessTransport(String... processCommandsAndArgs) {
+        if (processCommandsAndArgs.length == 0) {
+            throw new IllegalArgumentException("processCommandsAndArgs must not be empty");
+        }
+        this.processCommandsAndArgs = processCommandsAndArgs;
+    }
+    @Override
+    public String pass(String request) throws IOException {
+        final ProcessBuilder builder = new ProcessBuilder(processCommandsAndArgs);
+        final Process process = builder.start();
+        process.onExit().thenAccept((exitCode) -> {
+            System.out.println("Process exited with code: " + exitCode);
```

Comment:
Consider using com.intellij.openapi.diagnostic.Logger instead of System.out.println. This is just a note for the future and not this PR.